src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Button({ name }) {
```

Comment:
### [Optional]

For good naming convention, it would be great to use `buttonName` instead of just `name`